spec/api/v2/schemas.rb

```diff
@@ -6,13 +6,17 @@ module Api
   module V2
     module Schemas
       include Metadata
+      include Policies
```

Comment:
Please extract this into a separate PR, I understand it is required, but still out of scope.

Reply:
@romanblanco If you agree, I'd copy this into my [rswag PR](https://github.com/RedHatInsights/compliance-backend/pull/1868) for now. There it wouldn't be out of scope.

Reply:
@RoamingNoMaD Yes, that would be better :+1: